lib/translator/translation.cc

```diff
@@ -39,7 +40,10 @@ BeginResponse Translation::Begin(absl::Span<const uint8_t> scsi_cmd,
   scsi_defs::OpCode opc = static_cast<scsi_defs::OpCode>(scsi_cmd[0]);
   switch (opc) {
     case scsi_defs::OpCode::kInquiry:
-      break;
+      pipeline_status_ = InquiryToNvme(scsi_cmd_, nvme_cmds_[0], nvme_cmds_[1],
+                                       response.alloc_len, lun);
+      nvme_cmd_count_ = 2;
+      return response;
```

Comment:
Is it necessary to `return` here, or could we `break` like the other cases and keep the control flow simpler?